cinema/views.py

```diff
@@ -127,6 +128,29 @@ def upload_image(self, request, pk=None):
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
In the upload_image method, you're returning a 400 BAD REQUEST response if the serializer is not valid. This is a good practice as it provides the client with useful error messages. However, you might want to consider logging these errors on the server side for debugging purposes.